lib/core/services/alfresco-api.service.ts

```diff
@@ -115,6 +118,8 @@ export class AlfrescoApiService {
             oauth.redirectUriLogout = window.location.origin + (oauth.redirectUriLogout || '/');
         }
 
+        this.storageService.storagePrefix = this.appConfig.get<string>(AppConfigValues.STORAGE_PREFIX, '');
```

Comment:
I think this would be better outside of this function. The method's name says initAlfrescoApi, which it does indeed. However now this method has a sideeffect, it initialises the Storage Service also. What do you think about putting this initialisation into the `load` method, just before initialising the alfresco-api?